### ISSUE TITLE
a11y keyboard navigation and screen reader improvements

### ISSUE THREAD

**Issue by wayne-shih** [shuding/nextra#3721]
Some accessibility fixes I'm planning on patching when I've got some time, jotting it down as a start... 

- [ ] Skip to content for keyboard navigation (something like [GDS](https://design-system.service.gov.uk/components/skip-link/))
- [ ] Missing aria-label 
    - [ ] Search box input
    - [ ] Breadcrumb (follow [NHS](https://service-manual.nhs.uk/design-system/components/breadcrumbs) / [APG](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/)) -- also change `div` to `nav`
- [ ] SVGs add `role="presentation"` when pre-described by other content 
    - [ ] Top nav grouped dropdown, chevron icon
    - [ ] Light/dark mode icon 
    - [ ] Sidebar collapse icon (more investigation needed to see if this is actually required) 
- [ ] ARIA hidden element must not be focusable or contain focusable elements
    - [ ] "Scroll to top"

**Comment by dimaMachina** [comment]
Happy to accept PR! Loved your previous contribution 😄

**Comment by dimaMachina** [comment]
> Skip to content 

skip to content was moved to `nextra/components` in v4 and is built in with `nextra-theme-docs` v4 https://github.com/shuding/nextra/blob/c9db20fb5667a0accb6ad248f63ae0051a643565/packages/nextra-theme-docs/src/layout.tsx#L102

**Comment by wayne-shih** [comment]
> > Skip to content
> 
> skip to content was moved to `nextra/components` in v4 and is built in with `nextra-theme-docs` v4
> 
> https://github.com/shuding/nextra/blob/c9db20fb5667a0accb6ad248f63ae0051a643565/packages/nextra-theme-docs/src/layout.tsx#L102

Ah nice! good to know, I'll cross that off my list for v3 fixes then. 

I haven't tried v4 yet -- will have a look at that too. Are you guys aiming towards a stable release data for v4?

**Comment by dimaMachina** [comment]
> I haven't tried v4 yet -- will have a look at that too. Are you guys aiming towards a stable release data for v4?

yes, first blog post with migration guide and 4.rc0, and after couple of weeks stable release

**Comment by dimaMachina** [comment]
@wayne-shih Nextra 4 is released, feel free to contribute

**Comment by wayne-shih** [comment]
I saw! Loving the heading target state animation... snazzy! 

This has gotten pushed down on my todo list, hopefully will have some free time next month or so to patch em up
